crunchy-bridge-for-analytics/results/crunchy-bridge-analytics-256.json

```diff
@@ -0,0 +1,55 @@
+{
+    "system": "Crunchy Bridge for Analytics (Parquet)",
+    "date": "2024-06-05",
+    "machine": "Crunchy Bridge (Analytics-256GB)",
```

Comment:
The "Machine" facet at https://benchmark.clickhouse.com/ usually refers to EC2 instance types or a `[CPU, RAM]` pair (except for t-shirt sizes 'M', 'S' etc. for ByteHouse and Snowflake who don't reveal the underlying hardware. "Analytics-256" does not immediately make the used hardware clear. I went through the steps described in README.txt of this PR and find that at some point "64 vCores / 256 GB DRAM" was shown for "Analytics-256". Could we therefore write `"machine": "Analytics-256GB (64 vCores, 256 GB)"` in l. 4?

Reply:
sure, I can update this.

Reply:
Thinking about it, results would be more comparable overall if they were taken on a smaller machine, e.g. something equivalent to c6a.4xlarge (only if this is possible).